documentation/staging/content/faq/node-selector.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Schedule pods to particular nodes"
+date: 2020-06-30T08:55:00-05:00
+draft: false
+weight: 15
+description: "How do I constrain scheduling WebLogic Server pods to particular nodes."
+---
+
+> How do I constrain scheduling WebLogic Server pods to a particular set of nodes?
+
```

Comment:
First step is to actually set a label on the nodes where the WebLogic Server pods will run.
For example
kubectl label nodes <your-node-name> name=weblogic-pods